### DISCUSSION TITLE
Add The functional role of this IP

### DISCUSSION THREAD

**Discussion by Shamil9310** [netbox-community/netbox#13498] [Ideas]
Good afternoon, please add to the line, The functional role of this IP "Unicast"

**Comment by bitcollector1** [comment]
I'm pretty sure they will just tell you to use tags or custom fields

**Comment by candlerb** [comment]
IP Address Status is already fully customisable: see https://docs.netbox.dev/en/stable/configuration/data-validation/#field_choices

IP Address Role is not (AFAIK), but the implication when the role is unset is that it's a normal address, i.e. unicast.
